index.js

```diff
@@ -1,6 +1,13 @@
 "use strict";
 var config = require("./config");
+var mongoose = require('mongoose');
 var ContribCat = require("./lib");
 var contribCat = new ContribCat(config);
-contribCat.run().then(contribCat.runReporters.bind(contribCat));
 
+mongoose.connect("mongodb://localhost/contribcat");
```

Comment:
lets make this a config variable? and all refs to mongoose out of index and server and into lib - given everything else in lib depends on it anyway.